File: backend/services/integration-service/src/main/java/com/lmspilot/integration/IntegrationServiceApplication.java
package com.lmspilot.integration;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class IntegrationServiceApplication {
    public static void main(String[] args) { SpringApplication.run(IntegrationServiceApplication.class, args); }
}

File: backend/services/integration-service/src/main/kotlin/com/lmspilot/integration/IntegrationServiceApplication.kt
package com.lmspilot.integration

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class IntegrationServiceApplication

fun main(args: Array<String>) {
    runApplication<IntegrationServiceApplication>(*args)
}
